scripts/monthly_means.py

```diff
@@ -49,12 +58,14 @@ def process_month(files: list[Path], product: str) -> xr.Dataset:
         time_deltas = [i * np.timedelta64(3, 'h') for i in range(8)]
     else:
         time_deltas = [i * np.timedelta64(30, 'm') for i in range(24 * 2)]
-    
+
     # .astype('datetime64[M]').astype('datetime64[m]'):
     # hack to avoid dealing with days, i.e. date set to YYYYmmddT00:00
     time_offset = ds.time.values[0].astype('datetime64[M]').astype('datetime64[m]')
-    time_values = [time_offset + delta for delta in time_deltas]
-    
+    time_values = np.array([time_offset + delta for delta in time_deltas])
+    time_bounds = time_values[:-1] + 0.5 * (time_values[1:] - time_values[:-1])
```

Comment:
More elegant as `(time_values[:-1] + time_values[1:]) / 2`? In any case, it's good.